Application/Education/Certificate/Generator/Service/DataCertificate/SDataGym.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+
+namespace SPHERE\Application\Education\Certificate\Generator\Service\DataCertificate;
+
+
+use SPHERE\Application\Education\Certificate\Generator\Service\Data;
+use SPHERE\Application\Education\Lesson\Division\Division;
+
+class SDataGym
+{
+
+    public static function setCertificateStandard(Data $Data)
+    {
+
+        self::setGymHjInfo($Data);
+        self::setGymHj($Data);
+        self::setGymJ($Data);
+        self::setGymKurshalbjahreszeugnis($Data);
+        self::setGymAbitur($Data);
+        self::setGymAbgSekI($Data);
+        self::setGymAbgSekII($Data);
+    }
+
+    /**
+     * @param Data $Data
+     */
+    private static function setGymHjInfo(Data $Data)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setGymHjInfo() has an NPath complexity of 621. The configured NPath complexity threshold is 200.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)